website/client/js/controllers/menuCtrl.js

```diff
@@ -13,7 +13,8 @@ angular.module('habitrpg')
         var user = $scope.user;
         if (user.purchased && user.purchased.plan && user.purchased.plan.mysteryItems && user.purchased.plan.mysteryItems.length) {
           return mysteryValue;
-        } else if ((user.invitations.party && user.invitations.party.id) || (user.invitations.guilds && user.invitations.guilds.length > 0)) {
+        } else if ((user.invitations.party && user.invitations.party.id) || (user.invitations.guilds && user.invitations.guilds.length > 0)
```

Comment:
line was longer before this change, it's okay and we're not enforcing a strong code style on client side code